Makefile.work

```diff
@@ -185,8 +185,10 @@ ifneq ($(SONIC_DPKG_CACHE_SOURCE),)
 endif
 
 ifeq ($(SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD), y)
+ifneq ($(MULTIARCH_QEMU_ENVIRON), y)
```

Comment:
Does MULTIARCH_QEMU_ENVIRON support native docker? If not, can we simply print a wrong usage if MULTIARCH_QEMU_ENVIRON set to y.

Reply:
For MULTIARCH_QEMU_ENVIRON=y, native docker will always be used.

DOCKER_RUN += -v /var/run/march/docker.sock:/var/run/docker.sock

Reply:
I don't entirely understand the current multiarch code, but from what I can tell, there are actually two native dockerd instances being started up. The first one is for iptables rules to get applied, and the second one is to load the `docker-*.gz` container files. I don't know why there's two separate dockerd instances here and why the one that is already running natively can't be used.